lib/src/logging.dart

```diff
@@ -0,0 +1,71 @@
+// ignore_for_file: avoid_print
+
+import 'package:logging/logging.dart' as logging;
+
+final _activeLoggers = <logging.Logger>{};
+
+/// Defines the names used for the loggers.
+class LogNames {
+  static const leader = 'leader';
+  static const follower = 'follower';
+}
+
+final leaderLog = logging.Logger(LogNames.leader);
```

Comment:
FYI, these logs should be in a class, too, so we would say `ftlLogs.leader.fine()`. That's because there could be other projects with logs that have the same name. If so, we'd have to qualify every import, which is a paint.

I'll make this adjustment on my end because I'm gonna work on a new API based on our other conversations.